firefox-plugins-installer

```diff
@@ -43,11 +43,11 @@ JAVA_LINK=${PLUGINS_DIR}/libnpjp2.so
 # previous versions are often removed from the server, so it is safest
 # for an automated script to use the stable version in the archive
 # http://fpdownload.macromedia.com/get/flashplayer/pdc/11.2.202.643/install_flash_player_11_linux.x86_64.tar.gz
-FLASH_VERSION=11.2.202.644
-FLASH_SHA256SUM=aba0e2f5940fc2588357e93f7bad84a0db51792b2f2bd1d24be9ef491e87c041
-FLASH_FILENAME=fp_${FLASH_VERSION}_archive.zip
+FLASH_VERSION=24.0.0.221
```

Comment:
This will break as soon as Adobe decides to ship newer version of the plugin, as they tend to remove the previous version from the servers.

Maybe we could leverage the LKGV file in https://s3-us-west-2.amazonaws.com/abacadaba/flashplugin-x86_64.lkgv for this too, and add the necessary code here to check it and build the actual URL based on it? (replacing `ppapi` with 
`npapi` to get the right URL for firefox).

Or even, if we want to make it more independent from the PPAPI plugin, we should extend the flashplugin-x86_64.lkgv file so that we include there the latest versions of the PPAPI **and** the NPAPI plugin, along with the extra data (URL, SHA256SUM), and also update the [jenkins-check-adobe-flash-plugin script](https://github.com/endlessm/eos-build/blob/master/ci-integration/jenkins-check-adobe-flash-plugin) to check for changes in both versions too. That way it should be more robust and easier to handle in the future, without requiring OS upgrades.